src/main/res/values/strings.xml

```diff
@@ -1003,7 +1003,9 @@
     <!-- placeholder will be replaced by the name of the inviter. -->
     <string name="secure_join_replies">%1$s replied, waiting for being added to the group…</string>
     <string name="secure_join_wait">Establishing guaranteed end-to-end encryption, please wait…</string>
+    <!-- deprecated -->
     <string name="secure_join_wait_timeout">Could not yet establish guaranteed end-to-end encryption, but you may already send a message.</string>
+    <string name="secure_join_takes_longer">This takes longer than expected, maybe devices are offline…\n\nHowever, the process continues in background, you can do something else 🕺</string>
```

Comment:
After the discussion in the core PR i suggest "maybe the contact or you are offline". This can guide the user to check their network i think. "Devices" may be not clear enough (whose devices are meant?)

Reply:
I think "This is taking longer than expected" is more correct than "this takes longer than expected"

Reply:
thanks for the suggestions, i did a commit!

for the "than expected": i stroke "than expected", this is not an error, we're offline first. it _is_ expected that it can take hours, _therefore_ we continues in background